README.md

```diff
@@ -97,7 +97,7 @@ network node. It provides the base APIs `config_set`, `config_get`, and
 
 ### CommandReference
 
-The `CommandReference` module provides for the abstraction of NX-OS CLI,
+The `CommandReference` class provides for the abstraction of NX-OS CLI,
```

Comment:
Should we keep these 'NX-OS' references ? There are several throughout the code that seem out of place now.